src/Hyperion/Main.hs

```diff
@@ -130,22 +134,26 @@ doRun bks = do
       throwIO $ DuplicateNames [ n | n:_:_ <- group (sort nms) ]
     foldMap (runBenchmark (sample 100 (fixed 5))) bks
 
-doAnalyze :: Config -> [Benchmark] -> IO ()
-doAnalyze Config{..} bks = do
+doAnalyze :: Config -> Text -> [Benchmark] -> IO ()
+doAnalyze Config{..} benchName bks = do
+    packageName <- getProgName
```

Comment:
You are right the name is the one of the executable containing the bench.

This is done so that one can have multiple benches in an executable, and multiple executables in a same project.

If you have a better naming idea I'm all up for it :)